CHANGES.rst

```diff
@@ -1248,6 +1248,10 @@ astropy.stats
 astropy.table
 ^^^^^^^^^^^^^
 
+- Fixed a bug where ``int64`` get converted to float and lose precision when
```

Comment:
Not just `int64`, but any `int` or `unsigned int`.  How about:
```
- Fixed a bug in ``to_pandas()`` where integer type masked columns were always getting
  converted to float.  This could cause loss of precision.  Now this only occurs if there are 
  actually masked data values, in which case pandas does require the values to be float 
  so that ``Nan`` can be used to mark the masked values.
```